cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -58,6 +58,9 @@ type MasterConfiguration struct {
 	ImageRepository string
 	// UnifiedControlPlaneImage specifies if a specific container image should be used for all control plane components
 	UnifiedControlPlaneImage string
+
+	// FeatureFlags enabled by the user
+	FeatureFlags []string
```

Comment:
Done. Because it's now specified with a map, I removed the flagset addition - users will need to specify the feature flags in the config file. Is that okay with you?